common/setups/rasr/util/flow/tensorflow.py

```diff
@@ -0,0 +1,86 @@
+__all__ = ["get_tf_flow", "add_tf_flow_to_base_flow"]
+
+from typing import List, Optional, Union
+
+from sisyphus import tk
+
+import i6_core.rasr as rasr
+import i6_core.returnn as returnn
+
+
+def get_tf_flow(
+    checkpoint_path: Union[tk.Path, returnn.Checkpoint],
+    tf_graph_path: tk.Path,
+    returnn_op_path: Optional[Union[tk.Path, List[tk.Path]]] = None,
+    forward_output_layer: str = "log_output",
+    tf_fwd_input_name: str = "tf-fwd-input",
+) -> rasr.FlowNetwork:
+    """
+    Create flow network and config for the tf-fwd node
+    """
+    tf_flow = rasr.FlowNetwork()
+
+    tf_flow.add_input(tf_fwd_input_name)
+    tf_flow.add_output("features")
+    tf_flow.add_param("id")
+    tf_fwd = tf_flow.add_node("tensorflow-forward", "tf-fwd", {"id": "$(id)"})
+    tf_flow.link(f"network:{tf_fwd_input_name}", tf_fwd + ":input")
+    tf_flow.link(tf_fwd + ":log-posteriors", "network:features")
+
+    tf_flow.config = rasr.RasrConfig()
+
+    tf_flow.config[tf_fwd].input_map.info_0.param_name = "input"
+    tf_flow.config[
+        tf_fwd
+    ].input_map.info_0.tensor_name = "extern_data/placeholders/data/data"
```

Comment:
Is it good to have such hardcoded assumption on the input data name?

Reply:
No, see https://github.com/rwth-i6/i6_core/pull/307/